lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -135,12 +136,16 @@ def join_bulk_responses(bulk_responses)
     def bulk_send(body_stream)
       params = http_compression ? {:headers => {"Content-Encoding" => "gzip"}} : {}
       # Discard the URL 
+      @metric.increment(:bulk_requests)
+      @metric.increment(:concurrent_bulk_requests)
```

Comment:
Here's my thought on what a user might want to know:

* Is the ES output stuck?
* Is the ES output slow?

For these questions, concurrent_bulk_requests (CBS) doesn't tell us enough, partly because you need to know more context (Is CBS always max because ES is our main bottleneck? Are event counts going up?)

In terms of the questions I proposed above --

* Is ES stuck?

We can infer this by observing two values today: `in` vs `out` count for the ES output being nonzero *and* having both values stable (not increasing). Across two observations (of reasonable time, say, 10 seconds), if `in` is higher than `out` and neither value is growing, then we know ES output is waiting on Elasticsearch.

* Is ES slow?

We can infer this by observing the `duration_in_millis`/`out` to compute per-event latency through Elasticsearch. If the event latency is higher than expected, then Elasticsearch output is a bottleneck.